src/cdk-experimental/listbox/listbox.ts

```diff
@@ -194,360 +158,382 @@ export class CdkOption<T = unknown> implements ListKeyManagerOption, Highlightab
-    const options = this._options;
-
-    return options.changes.pipe(
-      startWith(options),
-      switchMap(() => merge(...options.map(option => option.selectionChange))),
-    );
-  }) as Observable<OptionSelectionChangeEvent<T>>;
-
-  private _disabled: boolean = false;
-  private _multiple: boolean = false;
-  private _useActiveDescendant: boolean = false;
-  private _autoFocus: boolean = true;
-  private _activeOption: CdkOption<T>;
-  private readonly _destroyed = new Subject<void>();
-
-  @ContentChildren(CdkOption, {descendants: true}) _options: QueryList<CdkOption<T>>;
-
-  @Output() readonly selectionChange = new EventEmitter<ListboxSelectionChangeEvent<T>>();
+export class CdkListbox<T = unknown> implements AfterContentInit, OnDestroy, ControlValueAccessor {
+  /** The id of the option's host element. */
+  @Input() id = `cdk-listbox-${nextListboxId++}`;
 
-  @Input() id = `cdk-listbox-${listboxId++}`;
+  // TODO(mmalerba): Should we normalize the order? Do we care about duplicates?
+  /** The value selected in the listbox, represented as an array of option values. */
+  @Input('cdkListboxValue')
+  get value(): T[] {
+    return this.selectionModel().selected;
+  }
+  set value(value: T[]) {
+    this.selectionModel().setSelection(...coerceArray(value == null ? [] : value));
+  }
 
   /**
-   * Whether the listbox allows multiple options to be selected.
-   * If `multiple` switches from `true` to `false`, all options are deselected.
+   * Whether the listbox allows multiple options to be selected. If the value switches from `true`
+   * to `false`, and more than one option is selected, all options are deselected.
    */
-  @Input()
+  @Input('cdkListboxMultiple')
   get multiple(): boolean {
     return this._multiple;
   }
   set multiple(value: BooleanInput) {
-    const coercedValue = coerceBooleanProperty(value);
-    this._updateSelectionOnMultiSelectionChange(coercedValue);
-    this._multiple = coercedValue;
+    this._multiple = coerceBooleanProperty(value);
+    this._updateSelectionMode();
   }
+  private _multiple: boolean = false;
 
-  @Input()
+  /** Whether the listbox is disabled. */
+  @Input('cdkListboxDisabled')
   get disabled(): boolean {
     return this._disabled;
   }
   set disabled(value: BooleanInput) {
     this._disabled = coerceBooleanProperty(value);
   }
+  private _disabled: boolean = false;
 
   /** Whether the listbox will use active descendant or will move focus onto the options. */
-  @Input()
+  @Input('cdkListboxUseActiveDescendant')
   get useActiveDescendant(): boolean {
     return this._useActiveDescendant;
   }
   set useActiveDescendant(shouldUseActiveDescendant: BooleanInput) {
     this._useActiveDescendant = coerceBooleanProperty(shouldUseActiveDescendant);
   }
+  private _useActiveDescendant: boolean = false;
 
+  // TODO(mmalerba): Why do we have this? Shouldn't it depend on whether we're using active descendant?
```

Comment:
I don't quite understand what the intent of this input was @jelbourn do you remember?

Reply:
I'm guessing in case people want to use actual focus instead of `aria-activedescendant`. I think that we can drop it and add it back in if users ask for it. Right now it's just adding complexity.

Reply:
I *think* maybe the idea here was that you could set `autoFocus=false activeDescendant=false` and then it would treat both the listbox *and* the current active option as separate tab stops. I'm not sure that really makes any sense though, so I think getting rid of it is the right call. Unless anyone can think of a reason why that would be desirable behavior